app/templates/notifications.hbs

```diff
@@ -0,0 +1,10 @@
+<h1 class="ui header">{{t 'Notifications'}}</h1>
+<div class="ui divider"></div>
+<div class="row">
+  <a href="/notifications" class="ui button left attached">{{t 'Unread'}}</a>
```

Comment:
@geekyd Its the property of semantic's attached buttons I think and in my opinion it looks better than 👇
<img width="241" alt="screen shot 2017-06-02 at 2 14 09 pm" src="https://cloud.githubusercontent.com/assets/12716067/26718194/cd94f1ec-479d-11e7-9c29-86ef3146c41d.png">